templates/explore.html

```diff
@@ -32,6 +32,24 @@
             </div>
         </form>
     </div>
+    <div class="w-full mb-4" id="program_filter">
+        <form method="GET">
+            <div class="flex flex-row px-4 w-full items-center border border-gray-400 py-2 rounded-lg gap-4">
+                <custom-select name="adventure" id="explore_page_adventure" data-autosubmit="true" class="w-2/3" data-type="single">
+                    <option hidden {% if not filtered_adventure %} selected{% endif %} value="">&mdash; {{ _('adventure') }} &mdash;</option>
+                    {% for adventure_key, name in adventures_names.items() %}
+                        <option hidden value="{{ adventure_key }}" {% if adventure_key == filtered_adventure %}selected{% endif %}>{{ name }}</option>
+                    {% endfor %}
+                </custom-select>
+                <custom-select name="level" id="explore_page_level" data-autosubmit="true" class="w-1/3" data-type="single">
+                    <option hidden {% if not filtered_adventure %}selected{% endif %} value="">&mdash; {{_('level')}} &mdash;</option>
+                    {% for i in range(1, (max_level + 1)) %}
+                        <option hidden value="{{ i }}" {% if filtered_level and i|string == filtered_level %}selected{% endif %}>{{ i }}</option>
+                    {% endfor %}                
+                </custom-select>
+            </div>
+        </form>
+    </div>
```

Comment:
- still see the old dropdowns
- if i click on the option in the new select custom elements, the filtering isn't reflected on the ui
![image](https://github.com/hedyorg/hedy/assets/20051470/9208c8f3-74e5-478c-aa57-90303b4b0e55)

Reply:
Yeah, thanks for pointing this out! Fixed it with HTMX 😄